src/picosoc/constr/basys3-nextpnr-xilinx.xdc

```diff
@@ -0,0 +1,80 @@
+# Clock Signal
```

Comment:
Same thing as above. Is this needed for nextpnr-xilinx only and it does not work for other toolchains?

Reply:
Same as above.